test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj

```diff
@@ -31,20 +31,22 @@
 
 (deftest ^:integration legacy-routes
   (testing "The legacy web routing service properly handles old routes."
-    (bootstrap/with-puppetserver-running app
-      {}
-      (is (= 200 (:status (http-get "/v2.0/environments"))))
-      (is (= 200 (:status (http-get "/production/node/localhost"))))
-      (is (= 200 (:status (http-get "/production/certificate_statuses/all")))))))
+    (bootstrap/with-puppetserver-running-with-mock-jrubies
+     app
+     {}
+     (is (= 200 (:status (http-get "/v2.0/environments"))))
+     (is (= 200 (:status (http-get "/production/node/localhost"))))
+     (is (= 200 (:status (http-get "/production/certificate_statuses/all")))))))
```

Comment:
@cprice404 I don't think any other acceptance tests cover it.  I agree that we should add some more assertions beyond just the status code - some that validate the content of the response body.  The part I'm still a little on the fence about is whether we think we could use mock JRubies / maybe in combination with something that echoes the request map supplied to `.handleRequest` back out through the response -- so we can assert that the route was translated correctly -- or if we think we need a "real" JRubyPuppet so we can get back "real" indirector endpoint responses.  Probably just need to make a call one way or the other on that.  I can go either way.  I'm comfortable with the mock approach for speed but understand if you're concerned enough about insufficient real test coverage that we should instead use real JRubies.  What do you think?